dax/computer/service/computer.go

```diff
@@ -123,27 +119,21 @@ type CommandConfig struct {
 }
 
 func newCommand(addr dax.Address, cfg CommandConfig) *fbserver.Command {
-	// Set up WriteLogger.
-	// TODO(tlt): since WriteLogger is no longer a separate service (but
-	// rather just a directory path) its configuration could be moved under
-	// computer, and then get rid of WriteLogger.Run. This would become "if
-	// DataDir != ''". Let's do this after we get rid of the dax integration
-	// tests which start up separate writelogger and snapshotter containers.
-	var wlSvc *writelogger.WriteLogger
-	if cfg.WriteLoggerRun {
-		wlSvc = writelogger.New(writelogger.Config{
-			DataDir: cfg.WriteLoggerConfig.DataDir,
-			Logger:  cfg.Logger,
-		})
+	var wlSvc computer.WritelogService
+	if cfg.ComputerConfig.WriteloggerDir != "" {
+		wlSvc = writelogger.New(cfg.ComputerConfig.WriteloggerDir, cfg.Logger)
+	} else {
+		wlSvc = computer.NewNopWritelogService()
+		cfg.Logger.Warnf("No writelogger configured, dynamic scaling will not function properly.")
 	}
 
 	// Set up Snapshotter.
-	var ssSvc *snapshotter.Snapshotter
-	if cfg.SnapshotterRun {
-		ssSvc = snapshotter.New(snapshotter.Config{
-			DataDir: cfg.SnapshotterConfig.DataDir,
-			Logger:  cfg.Logger,
-		})
+	var ssSvc computer.SnapshotService
+	if cfg.ComputerConfig.SnapshotterDir != "" {
+		ssSvc = snapshotter.New(cfg.ComputerConfig.SnapshotterDir, cfg.Logger)
+	} else {
+		ssSvc = computer.NewNopSnapshotterService()
+		cfg.Logger.Warnf("No snapshotter configured.")
```

Comment:
if you want a no-op you should have to explicitly ask for it so we don't have folks accidentally running w/o snapshot/writelog and not knowing it.

...and by folks I mean us

Reply:
I changed this to accept `"NULL"` or error on `""`

Reply:
changed it to case-insensitive "off"